### PR TITLE
feat(providers): Return http error when network is unsupported #44

### PR DESCRIPTION
**Pull request by Mitsichury**

### CODE REVIEW

File: src/client/getProviderUrl.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const hostByChainId: Record<number, string> =
```
Comment (dmosites) [inline src/client/getProviderUrl.ts:1]: Recommend using `apiUrls` from `@airswap/constants` for these.
https://github.com/airswap/airswap-protocols/blob/develop/tools/constants/index.ts#L85

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   "type": "module",
   "license": "MIT",
   "dependencies": {
-    "@airswap/libraries": "4.0.10",
+    "@airswap/libraries": "4.0.13",
```
Comment (dmosites) [inline package.json:19]: Why not latest `4.0.19`?

File: src/client/getProviderUrl.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    42220: "https://celo-mainnet.infura.io/v3",
+    44787: "https://celo-alfajores.infura.io/v3"
+}
+
+export function getProviderUrl(chainId: number, apiKey: string) {
+    const host = hostByChainId[chainId];
+    if (!host) {
+        throw new Error("Unknown chain ID");
+    }
+    return `https://${host}/${apiKey}`;
```
Comment (dmosites) [inline src/client/getProviderUrl.ts:29]: Only append apiKey if host URL includes `infura.io/v3`.

**Review by dmosites — CHANGES_REQUESTED** [review]

File: src/client/getProviderUrl.ts (hunk 3 above)
Comment (Mitsichury) [reply src/client/getProviderUrl.ts]: yes thank you

File: package.json (hunk 2 above)
Comment (Mitsichury) [reply package.json]: done

File: src/client/getProviderUrl.ts (hunk 1 above)
Comment (Mitsichury) [reply src/client/getProviderUrl.ts:3]: using lib and fallback now, thanks :)

Comment (dmosites) [reply src/client/getProviderUrl.ts:3]: Please remove the fallback as AirSwap only supports networks found in `apiUrls`.

**Review by dmosites — APPROVED** [review]

**Pull request merged**